File: tests/fixtures/real_index/python_project/app/service.py
"""Cross-file references to both identically named functions."""

from app.auth import session_token as auth_token
from app.billing import session_token as billing_token


def build(user: str, plan: str) -> str:
    total = f"日本語 {auth_token(user)}"
    return total + "|" + billing_token(plan)

File: tests/fixtures/real_index/typescript_project/src/service.ts
import { sessionToken as authToken } from './auth'
import { sessionToken as billingToken } from './billing'
import { DiskStore, MemoryStore, Store } from './store'

export function build(user: string, plan: string): string {
  const total = `日本語 ${authToken(user)}`
  return `${total}|${billingToken(plan)}`
}

export function stores(): Store[] {
  return [new MemoryStore(), new DiskStore()]
}
